static/styles/dark_theme.css

```diff
@@ -521,6 +521,7 @@ body.dark-theme {
     #user_presences li:hover .user-list-sidebar-menu-icon,
     li.top_left_all_messages:hover .all-messages-sidebar-menu-icon,
     li.top_left_starred_messages:hover .starred-messages-sidebar-menu-icon,
+    li.top_left_drafts:hover .drafts-sidebar-menu-icon,
```

Comment:
The hover state of the drafts section was not working as intended compared to the All messages and stared massage section on the left sidebar, this caused an issue of UI inconsistency on the draft message section.

This commit fixed this by adding the class associated with the drafts section to CSS the code on user_presenses in dark_theme.css as it was missing that line of code.